static/src/javascripts/projects/common/modules/experiments/tests/remote-render-epic.js

```diff
@@ -110,6 +110,11 @@ const remoteRenderTest = {
                     isMinuteArticle: config.hasTone('Minute'),
                     isPaidContent: page.isPaidContent,
                     tags: buildKeywordTags(page),
+                    // These are hardcoded as to no stop the Contributions service from sending the Epic.
```

Comment:
Maybe just make it explicit this is a very temporary 'fix'?

Reply:
Updated